### PR TITLE
Recordings e2e

### PR DESCRIPTION
**Pull request by streamer45**
#### Summary

It was a bit of journey to get these to work. One failure led to another and getting the job service to work in a docker in docker setup was slightly challenging and required a couple of adaptations (see related PRs) but it's now done and functioning.

#### Related PRs

https://git.internal.mattermost.com/qa/calls-e2e-testing/-/merge_requests/22

https://github.com/mattermost/calls-offloader/pull/16

https://github.com/mattermost/calls-recorder/pull/22

### CODE REVIEW

File: e2e/tests/recordings.spec.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        await page.locator('[data-testid=SendMessageButton]').click();
+
+        // very recording ended prompt renders correctly
+        await expect(page.getByTestId('calls-widget-banner-recording')).toBeVisible();
+        await expect(page.getByTestId('calls-widget-banner-recording')).toContainText('Recording has stopped. Processing...');
+
+        // verify recording file has been posted by the bot (assumes CRT enabled)
+        await page.locator('.post__body').last().locator('.ThreadFooter button.ReplyButton').click();
+        await expect(page.locator('.ThreadViewer').locator('.post__header').last()).toContainText('calls');
+        await expect(page.locator('.ThreadViewer').locator('.post__header').last()).toContainText('BOT');
```
Comment (cpoile) [inline e2e/tests/recordings.spec.ts:54]: just curious -- why do those have to be separate?

**Review by cpoile — APPROVED** [review]
This is awesome Claudio, thank you!

File: e2e/tests/recordings.spec.ts (hunk 1 above)
Comment (streamer45) [reply e2e/tests/recordings.spec.ts:54]: I am sure there's a better way of doing it but the expected strings are not one after the other so checking for `calls BOT` isn't going to work as there are likely other DOM elements between them.

Comment (cpoile) [reply e2e/tests/recordings.spec.ts:54]: Ah, gotcha, thanks.

**Pull request merged**